tc-messaging/src/main/java/com/tc/entity/LinearVoltronEntityMultiResponse.java

```diff
@@ -276,4 +280,11 @@ public synchronized boolean startAdding() {
     }
     return false;
   }
+  
+  private boolean immediateSend = false;
+  
+  @Override
+  public boolean shouldSend() {
```

Comment:
should this be synchronized?

Reply:
no.  the path it is used on is only for a the same thread.  It is only used when the server is in single threaded mod optimization so the thread that is setting it is accessing it.  I'll add a note about that.